src/Plato_MeshMapUtils.hpp

```diff
@@ -54,12 +54,8 @@ struct Points
 } // namespace Plato
 
 
-namespace ArborX
-{
-namespace Traits
-{
 template <>
-struct Access<Plato::Geometry::BoundingBoxes, PrimitivesTag>
+struct ArborX::AccessTraits<Plato::Geometry::BoundingBoxes, PrimitivesTag>
```

Comment:
I think you need to qualify
```suggestion
struct ArborX::AccessTraits<Plato::Geometry::BoundingBoxes, ArborX::PrimitivesTag>
```